### PR TITLE
Add support for Flex target and update to the latest NBGL use cases

### PR DESCRIPTION
**Pull request by jasonxh**
Standardize on the following NBGL use cases:
- `nbgl_useCaseHomeAndSettings`
- `nbgl_useCaseAddressReview`
- `nbgl_useCaseReview`
- `nbgl_useCaseReviewBlindSigning`

# Checklist

- [ ] App update process has been followed 
- [x] Target branch is `develop` 
- [x] Application version has been bumped

### CODE REVIEW

File: src/ui/nbgl_display.c [hunk 1]
```diff
@@ -33,58 +33,54 @@
...
+    blind_sign_ctx.tip_box = tip_box;
+    blind_sign_ctx.choice_callback = choice_callback;
+
     if (N_storage.settings.allow_blind_signing) {
         blind_sign_info();
     } else {
-        nbgl_useCaseChoice(&C_round_warning_64px,
-                           "Enable blind signing to\nauthorize this\noperation",
+        nbgl_useCaseChoice(&C_Warning_64px,
+                           "Enable blind signing to authorize this operation?",
```
Comment (kent-white) [inline src/ui/nbgl_display.c:80]: Any reason for this change. I imagine the `n was done for formatting on the smaller ledger device

**Review by kent-white — APPROVED** [review]
Text is fine either way. But if there wasn't a good reason to change it for the flex I would prefer leaving the text the same.

File: src/ui/nbgl_display.c (hunk 1 above)
Comment (jasonxh) [reply src/ui/nbgl_display.c:80]: Was trying to follow the guidelines from ledger docs, which encourages consistency across apps. The example screenshots were all using natural sentences without artificial line breaks. Also I noticed in some cases the artificial line breaks can look awkward on Flex, which has slightly different screen size than Stax.

Comment (lpascal-ledger) [reply src/ui/nbgl_display.c:80]: @kent-white @jasonxh is this discussion blocking or could we merge + deploy this new version of Aptos?

Comment (kent-white) [reply src/ui/nbgl_display.c:80]: no. lets merge @lpascal-ledger

**Pull request merged**